### ISSUE TITLE
Away physics will stop working in FP 11.2?

### ISSUE THREAD

**Issue by medec** [away3d/awayphysics-core-fp11#7]
"Update – Nov 22, 2011: Starting with Flash Player 11.2 and AIR 3.2, content targeting Flash Player 11 and AIR 3 (i.e., content using SWF version 13 and above) will not support the experimental Alchemy prototype."

http://blogs.adobe.com/flashplayer/2011/09/updates-from-the-lab.html

**Comment by muzerly** [comment]
Yes, Before the more details about Alchemy2 come out, I have to suspend update of awayphysics.

**Comment by solverit** [comment]
That if the alchemy2 will still not very soon? What are your plans for the physics?

**Comment by ringodotnl** [comment]
Not much we can say at this point, we are talking to Adobe for a solution and let you know asap.

**Comment by hayesmaker** [comment]
looking forward to hearing some news on this project

**Comment by jamieowen** [comment]
Now adobe have decided on their direction with alchemy & premium content. What is the future of away physics?

**Comment by muzerly** [comment]
of course, I'm working on the new version of awayphysics. It will be more powerful.

**Comment by ringodotnl** [comment]
We will make sure it will continue to work.

**Comment by jamieowen** [comment]
sounds excellent!  :)

**Comment by jamieowen** [comment]
Also, Is there any planned support for softbody dynamics and/or Spring Constraints for cloth/rope simulations?

**Comment by 62316e** [comment]
As i know flasscc is already out. Still its not working on ios. Any reason?
